src/applications/disability-benefits/all-claims/pages/ptsdWalkthroughChoice781.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Looks like this file already exists as a `.jsx`. Was this added back in as a `.js` accidentally?